src/main/java/com/dabsquared/gitlabjenkins/workflow/AcceptGitLabMergeRequestStep.java

```diff
@@ -89,7 +105,19 @@ protected Void run() throws Exception {
             return null;
         }
 
-        private void println(String message) {
+        private String getCommitMessage(MergeRequest mergeRequest) {
+        	if (!step.useMRDescription)
+        		return step.mergeCommitMessage;
+        	
+        	println("Merge description is: "+mergeRequest.getDescription());
```

Comment:
Should this be using the `LOGGER` object instead?

Reply:
This is calling the println method defined below, which will use LOGGER, or send output to listener if there is a task listener in use.

Reply:
Looking at it again, it seems like a debug comment that I wouldn't like to include (will output needless data to the job console). I removed it. Thanks for catching this.

Reply:
Oh ok! :) Didn't see that one.